tests/tests_app/cli/test_cmd_launch.py

```diff
@@ -0,0 +1,327 @@
+import signal
+import time
+from functools import partial
+from multiprocessing import Process
+from pathlib import Path
+from unittest import mock
+from unittest.mock import ANY, MagicMock, Mock
+
+from click.testing import CliRunner
+
+from lightning.app.core.queues import QueuingSystem
+from lightning.app.frontend.web import StaticWebFrontend
+from lightning.app.runners.runtime import load_app_from_file
+from lightning.app.testing.helpers import _RunIf, EmptyWork
+from lightning.app.utilities.network import find_free_network_port
+
+try:
+    from lightning.app.utilities.app_commands import run_app_commands
```

Comment:
how does this happen a function is missing?

Reply:
Removing